src/components/WebexSettings/Tabs.scss

```diff
@@ -0,0 +1,35 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-tabs {
+  width: 100%;
+
+  .tabs-list {
+    border-bottom: 1px solid $md-gray-30;
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0 1.2rem;
+
+    .tab {
+      &.active > button {
+        border-bottom: 2px solid black;
+        font-weight: bold;
+      }
+
+      button {
+        background: transparent;
+        border: none;
+        cursor: pointer;
+        font-family: "CiscoSansTT Regular", "Helvetica Neue", Helvetica, Arial, sans-serif;
+        font-size: 16px;
+        padding: 0 0.5rem;
+
+        &:not(last-child) {
```

Comment:
should be `:last-child`, but the button is always the last child of its parent `<li>`, so I guess the rule should go on `li` elements.